cli/users_test.go

```diff
@@ -21,7 +21,27 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
+const (
+	createCommand           = "create"
+	getCommand              = "get"
+	tokenCommand            = "token"
+	refreshtokenCommand     = "refreshtoken"
+	updateCommand           = "update"
+	profileCommand          = "profile"
+	resetPasswordReqCommand = "resetpasswordrequest"
+	resetPasswordCommand    = "resetpassword"
+	passwordCommand         = "password"
+	enableCommand           = "enable"
+	disableCommand          = "disable"
+	deleteCommand           = "delete"
+	channelsCommand         = "channels"
+	thingsCommand           = "things"
+	domainsCommand          = "domains"
+	groupsCommand           = "groups"
+)
```

Comment:
Move this to a separate `commands_test.go` file and use `Cmd` rather than  `Command` suffix. Move other cmd constants, such as `connCmd` here.